.vscode/tasks.json

```diff
@@ -70,11 +70,11 @@
             }
         },
         {
-            "label": "v5_hal Firmware WSL Upload",
+            "label": "v5_hal Firmware Upload",
             "type": "shell",
-            "command": "catkin_make v5_hal_firmware_wsl_upload",

```

Comment:
Same as above; keeping a WSL upload would be nice if possible

Reply:
fixed in latest commit